content/en/synthetics/troubleshooting/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Synthetics Troubleshooting
+kind: documentation
+description: Troubleshoot common Synthetics issues.
+further_reading:
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your Synthetics tests"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+- link: "synthetics/api_tests"
+  tag: "Documentation"
+  text: "Configure an API Test"
+---
+
+If you experience issues setting up or configuring Datadog Synthetics, use this information to begin troubleshooting. If you continue to have trouble, [contact our AWESOME support team][1].
+
+## API Tests
+
+### Failed requests on a working endpoint
+
+Sometimes you know you have a working endpoint because when you visit your website in a browser (or cURL it), you get `2xx` status code. But when setting an API test on this endpoints or when hitting `Test URL` to perform a fast test you get `5xx` or `4xx` status code.
+
+This happens because cURL automatically sets a `user-agent` as a request header, and this also automatically happens when you visit the website in your browser. Datadog API tests don't automatically set a `user-agent`. This can be a problem because some websites ban requests that don't have a `user-agent` set, which causes Datadog API tests to return a `5xx` or `4xx` status code.
```

Comment:
```suggestion
This happens because cURL automatically sets a `user-agent` as a request header (as your browser), but Datadog API tests don't automatically set one. 
This can be a problem because some websites ban requests that don't have a `user-agent` set, which causes Datadog API tests to return a `5xx` or `4xx` status code.
```